README.md

```diff
@@ -119,3 +119,55 @@ By default footnote anchors are expected to be surrounded by brackets. If it is
 ```
 --css '.fnanchor:before { content: "["; } .fnanchor:after { content: "]"; }'
 ```
+
+## Footnote extraction styles supported
+
+##### html: `class="footnote"`
```

Comment:
These should either be a `###` to be one more than header above it, or they should be formatted differently -- but being H5s is just weird.